include/sys/pci.h

```diff
@@ -76,11 +76,11 @@ typedef struct pci_bus_driver {
 } pci_bus_driver_t;
 
 typedef struct pci_bar {
-  device_t *owner; /* pci device owner of this bar */
-  size_t size;     /* identified size of this bar */
-  int rid;         /* BAR number in [0,PCI_BAR_MAX-1] */
-  unsigned type;   /* RT_IOPORTS or RT_MEMORY */
-  unsigned flags;  /* nothing or RF_PREFETACHBLE */
+  device_t *owner;   /* pci device owner of this bar */
+  size_t size;       /* identified size of this bar */
+  int rid;           /* BAR number in [0,PCI_BAR_MAX-1] */
+  res_type_t type;   /* RT_IOPORTS or RT_MEMORY */
+  bool prefetchable; /* states whether memory bar is prefetchable */
```

Comment:
Please do not introduce changes that are not required to fix a problem. For various reasons we prefer not to use booleans in the kernel. In case of flags `true` usually doesn't speak for itself as an argument to function, while `RF_PREFETCHABLE` does.

Reply:
Done.